### PR TITLE
exclude Visual Studio 2015 `.vs` working directory

### PR DESCRIPTION
**Pull request by ghuntley**
"The .vs directory is the new location where per-user solution-specific items are saved. Previously there was no directory where this information could be placed, so the IDE kept adding files alongside the
solution. It's finally a solution to a problem that's been here for a very long time." - @sharwell

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,9 @@ bld/
 [Oo]bj/
 msbuild.log
 
+# Visual Studio 2015
+.vs/
+
 # Roslyn stuff
```
Comment (sharwell) [inline .gitignore]: :bulb: The new exclusion above can *replace* the items currently listed under `Roslyn stuff`. Visual Studio "14" CTPs used to create a folder **{SolutionName}.sln.ide**, but starting with CTP 6 it creates a folder just called **.vs**.

**Comment by jkotas** [comment]
LGTM

**Comment by kangaroo** [comment]
LGTM.  Can we remove the Pre-CTP6 block now?  Or set some date for it to go away in the future?

**Comment by jkotas** [comment]
Identical change was made in corefx https://github.com/dotnet/corefx/pull/1620

**Pull request merged**